### DISCUSSION TITLE
Implement .NET Core

### DISCUSSION THREAD

**Discussion by maivanchuong95** [speechmatics/community#8] [General chat]
Hello everyone, currently I intend to build a ConsoleApp .NET real-time transcript with audio taken from mircrophone using speechmatics. Anyone have any ideas to help me?
Thank you.

**Comment by rosstspeech** [comment]
Hi @maivanchuong95, thanks for your message.
Although Speechmatics do not provide a supported .NET client there is the following repo which includes examples of providing real time transcription with the microphone.

- https://github.com/jrg1381/smrtapi.net
- https://www.nuget.org/packages/Speechmatics.Realtime.Client

This is based on our Virtual Appliance deployment, but details for the SaaS based authentication is available here: 

- https://docs.speechmatics.com/introduction/rt-guide
- https://docs.speechmatics.com/rt-api-ref

Hope this helps!

**Reply by dpomeranec** [reply]
is there any possible to add on the live transcribe web the new function "button as output as keyboard" so I can transfer the live transcribe to the third party such as office word, notepad, etc ?
